cfpb/consumerfinance.gov
# Adjust Simple-Chart block wagtail layout

## Additions

- Made a new layout for the Simple-Chart wagtail block

## Changes

- Slight text change in one of the Simple-Chart fields

## How to test this PR

1. Pull PR
2. `yarn build` -- `./runserver.sh`
3. Go to http://localhost:8000/admin/pages/4481/edit/
4. Add Simple-Chart block and notice new layout


## Screenshots

![image](https://user-images.githubusercontent.com/85513449/164774881-d00bacdf-6932-4f8c-a235-21119d5741e0.png)

![image](https://user-images.githubusercontent.com/85513449/164774898-670e8f04-436d-4d3a-874a-bb0313f61b19.png)

Resolves https://github.cfpb.gov/CFPB/super-regular/issues/383
